File: Java/1010.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double cod1, numP1, valorUnita1;
        double cod2, numP2, valorUnita2;
        double TOTAL;
        cod1 = sc.nextDouble();
        numP1 = sc.nextDouble();
        valorUnita1 = sc.nextDouble();
        cod2 = sc.nextDouble();
        numP2 = sc.nextDouble();
        valorUnita2 = sc.nextDouble();

        TOTAL = ((numP1 * valorUnita1) + (numP2 * valorUnita2));
           
        
        System.out.printf("VALOR A PAGAR: R$ %.2f\n",TOTAL);
    }
}

File: Python/1010.py
cod1 , numP1 ,valorP1 = list(map(float,input().split())) 
cod2 , numP2 , valorP2 = list(map(float,input().split()))

toTalP1 = numP1*valorP1
toTalP2 = numP2*valorP2

TotalPagar = toTalP1+toTalP2

print (f"VALOR A PAGAR: R$ {TotalPagar:.2f}")


